test/AutobahnTestApp/Startup.cs

```diff
@@ -22,8 +22,10 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 if (context.WebSockets.IsWebSocketRequest)
                 {
                     logger.LogInformation("Received WebSocket request");
-                    var webSocket = await context.WebSockets.AcceptWebSocketAsync();
-                    await Echo(webSocket);
+                    using (var webSocket = await context.WebSockets.AcceptWebSocketAsync())
```

Comment:
Can you move this dispose into the middleware?
https://github.com/aspnet/WebSockets/issues/182#issuecomment-306214086